converters/fromZigbee.js

```diff
@@ -531,7 +532,14 @@ const converters = {
                 result.color_options = {execute_if_off: ((msg.data.options & 1<<0) > 0)};
             }
 
-            return result;
+            // handle color property sync
+            // NOTE: this should the last thing we do so we would have processed all values
+            //       the assign here is important as we do not want to wipe out other attributes
+            if (colorSync) {
```

Comment:
To avoid these repetitive if/else, we can always call the color sync function and pass meta.options to it. Then also `const colorSync = (meta.options && meta.options.hasOwnProperty('color_sync')) ? (meta.options.color_sync == true) : true;` doesn't have to be duplicated in every converter.